charts/sigstore-prober/Chart.yaml

```diff
@@ -21,4 +21,4 @@ annotations:
   artifacthub.io/license: Apache-2.0
   artifacthub.io/images: |
     - name: sigstore-prober
-      image: ghcr.io/sigstore/scaffolding/prober@sha256:d16b6489228e30cccda4395086148375f2f1aff395aa0b66dc54cfd46cf6309f
+      image: ghcr.io/sigstore/scaffolding/prober@sha256:79a270c22743f49a7be56961d95642a7be8e5799bfd3a0f2f7752ad10bf28d2c # v0.4.8
```

Comment:
I think we actually want this sha instead (which is the non-arch specific one):
sha256:f2984afbeff4e51963d6103c5bdf69f6fb1750bebbc584e22cecf0c89a4afe27

https://github.com/sigstore/scaffolding/pkgs/container/scaffolding%2Fprober/42671649?tag=v0.4.8

I think the one above is for linux/s390x (if you click on the OS/Arch tab that's the one)

Also checked that just re-running the release created the same sha as expected :)

Reply:
Ah thanks for catching that! I updated and rebased the branch.